menas/ui/components/schema/schemaDetail.controller.js

```diff
@@ -83,32 +83,49 @@ sap.ui.define([
       });
     },
 
-    onExportPress: function (oEv) {
+    onExportStructPress: function (oEv) {
       const schema = this._model.getProperty("/currentSchema");
-      new SchemaRestDAO().downloadSchema(schema.name, schema.version)
+      new SchemaRestDAO().getSchemaStruct(schema.name, schema.version)
+        .then((data, status, request) => {
+          const mimeType = "application/json";
+          const extension = "json";
+          this._downloadFile(schema, data, mimeType, extension);
```

Comment:
/json/FirstTestSchema/14 -> returns non formated json
/json/FirstTestSchema/14?pretty=true -> returns nicely formated response in browser but when saved in file text is `[object Object]` literally

One of the ways to go would be
```suggestion
          this._downloadFile(schema, JSON.stringify(data), mimeType, extension);
```